File: intest.cpp
/*
    codechef.com - Practice problem - Easy
    Enormous input test - http://www.codechef.com/problems/INTEST
    Language - C++
    Juliana Correa
*/

#include <iostream>
    
using namespace std;        
int main() {

   int n, k, x, divs = 0;
   cin >> n >> k;
    
   for (int i=0; i < n; i++) 
   {
      cin >> x;
      if (!(x%k)) divs++;
   }      
   
   cout << divs << endl;    
   return 0;
} 

File: intest.f90
!    codechef.com - Practice problem - Easy
!    Enormous input test - http://www.codechef.com/problems/INTEST
!    Language - Fortran
!    Juliana Correa

program enormous
   implicit none

   integer :: n, k, x, i, divs
   read *, n, k

   i = 0
   divs = 0
   do while (i < n)
      read *, x
      if (mod(x,k) == 0) then
         divs = divs + 1
      end if
      i = i + 1
   end do

   print *, divs

end program
